stdlib/influxdata/influxdb/v1/v1.flux

```diff
@@ -34,6 +34,7 @@ tagKeys = (bucket, predicate=(r) => true, start=-30d) =>
         |> filter(fn: predicate)
         |> keys()
         |> keep(columns: ["_value"])
+        |> unique()
```

Comment:
This should probably be `distinct` and should be after the call to `keys`. If I remember right that is what the planner was expecting.

Reply:
What's the difference between unique and distinct?

Reply:
Unique is a selector, so it returns the first row for each new value. Distinct is more like an aggregate where all other data is lost except for the set of distinct values

Reply:
Ah. Do we need to have `keep` in there then? It seems to me like `distinct()` is the same as `keep(columns: ["_value"]) |> unique()`?

Reply:
I still think that this needs to be after the `keys` call and before the `keep` call, otherwise the planner won't know how to optimize. I could be remembering incorrectly 

I think we need the keep to remove some extraneous columns but I don't remember exactly.

Reply:
It doesn't work in the opposite order because keep also ends up grouping everything into the same table do distinct works.

This will be used by the new rpc call so we don't need to really worry about the old one.

Reply:
Another one that works group with no arguments and then distinct.